<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>To3D</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>965</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionAdd_Images"/>
    <addaction name="actionAdd_Video"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuModel">
    <property name="title">
     <string>Model</string>
    </property>
    <addaction name="actionExtract_Features"/>
    <addaction name="actionRecunstruct"/>
    <addaction name="separator"/>
    <addaction name="actionReload_3D_Model"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionFeedback"/>
    <addaction name="actionHelp"/>
    <addaction name="actionWebsite"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuModel"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_treeview">
   <property name="minimumSize">
    <size>
     <width>270</width>
     <height>35</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Model View</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QTreeView" name="treeView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>351</width>
       <height>311</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_viewer">
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>35</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Viewer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <widget class="QTabWidget" name="tabview_viewer">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>331</width>
       <height>341</height>
      </rect>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2d">
      <attribute name="title">
       <string>2D View</string>
      </attribute>
      <widget class="QLabel" name="viewer_2d">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>100</y>
         <width>161</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3d">
      <attribute name="title">
       <string>3D View</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_info">
      <attribute name="title">
       <string>Info</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_log">
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>170</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Logs</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <widget class="QPlainTextEdit" name="textbox_log">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>781</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdd_Images">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Add Images</string>
   </property>
  </action>
  <action name="actionAdd_Video">
   <property name="text">
    <string>Add Video</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionFeedback">
   <property name="text">
    <string>Feedback</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionWebsite">
   <property name="text">
    <string>Website</string>
   </property>
  </action>
  <action name="actionExtract_Features">
   <property name="text">
    <string>Extract Features</string>
   </property>
  </action>
  <action name="actionRecunstruct">
   <property name="text">
    <string>Recunstruct</string>
   </property>
  </action>
  <action name="actionReload_3D_Model">
   <property name="text">
    <string>Reload 3D Model</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
